Source/Core/DolphinTool/ConvertCommand.cpp

```diff
@@ -33,24 +33,24 @@ int ConvertCommand::Main(const std::vector<std::string>& args)
 
   parser->add_option("-s", "--scrub")
       .action("store_true")
-      .help("Scrub data as part of conversion.");
+      .help("Scrub junk data as part of conversion.");
 
   parser->add_option("-b", "--block_size")
       .type("int")
       .action("store")
-      .help("Block size for GCZ/WIA/RVZ formats. RVZ Default = 131072 (128KiB)");
+      .help("Block size for GCZ/WIA/RVZ formats, as an integer. RVZ suggested: 131072 (128KiB)");
 
-  parser->add_option("-c", "--compress")
+  parser->add_option("-c", "--compression")
       .type("string")
       .action("store")
-      .help("Compression method to use when converting to WIA/RVZ. Default is Zstandard (zstd). "
+      .help("Compression method to use when converting to WIA/RVZ. Suggested: Zstandard (zstd). "
```

Comment:
```suggestion
      .help("Compression method to use when converting to WIA/RVZ. RVZ suggested: Zstandard (zstd)."
```

Since WIA doesn't support zstd.